### ISSUE TITLE
change the value by code

### ISSUE THREAD

**Issue by wladimiravila** [valor-software/ng2-select#381]
hi guys, I'm trying to change the <ng2-select> value binding property 'data' or 'active'  without success, is possible set null o selected value from *component.ts? 

thanks for your help

**Comment by martijnpieters** [comment]
Yes, it's possible with a tricky solution: https://github.com/valor-software/ng2-select/issues/333#issuecomment-237915397
